source/marsGameHud.vwf.yaml

```diff
@@ -0,0 +1,468 @@
+---
+extends: http://vwf.example.com/hud/overlay.vwf
+implements:
+  - http://vwf.example.com/sceneGetter.vwf
+properties:
+  visible: true
+  lastBlocklyNodeID:
+methods:
+  elementPreDraw:
+    parameters:
+      - context
+      - element
+    body: |
+      var alpha = 1;
+      var time = Date.now() / 1000;
+      if ( !element.enabled ) {
+        alpha = 0.5;
+      } else if ( element.isBlinking ) {
+        if ( time  - element.lastBlinkTime > element.blinkInterval ) {
+          alpha = 0.5;
+          if ( time - element.lastBlinkTime > element.blinkInterval + element.blinkDuration ) {
+            element.lastBlinkTime = time;
+          }
+        }
+      }
+      context.globalAlpha = alpha;
+  elementPostDraw:
+    parameters:
+      - context
+      - element
+    body: |
+      context.globalAlpha = 1;
+  setAllBlinking:
+  setAllEnabled:
+children:
+  blocklyButton:
```

Comment:
Given that the children are moderately complex, would it make sense to give each child its own .js and .yaml files, maybe all in some subdirectory like source/hud (so we don't clutter up the source directory too much)?  I don't feel strongly for the HUD, but it seems like that might be a bit cleaner.